src/pydicom/uid.py

```diff
@@ -583,439 +583,397 @@ def generate_uid(
 # Only auto-generated Storage SOP Class UIDs below - do not edit manually
 
 
-MediaStorageDirectoryStorage = UID("1.2.840.10008.1.3.10")  # noqa
```

Comment:
had the generator been changed before to not create these comments?

Reply:
That was from #2101. I honestly would've been happier without the 10s of thousands of changed lines, but I guess the band aid is ripped off now.